AirCasting/SessionViews/SingleMeasurementView.swift

```diff
@@ -5,9 +5,17 @@ import SwiftUI
 import AirCastingStyling
 import Resolver
 
+class SingleMeasurementViewThreshold: ObservableObject {
+    var value: SensorThreshold? = nil
+    
+    init(value: SensorThreshold? = nil) {
+        self.value = value
+    }
+}
+
```

Comment:
You are double nil'ing here. First, you define the value as optional nil. After that, we will call the init and assing /default value/ from init to our `var value` again. If you need more explanation please let me know 😄